docs/evaluate/temporal-cloud/gcp-private-service-connect.mdx

```diff
@@ -17,4 +17,80 @@ tags:
   - private service connect
 ---
 
-Awaiting draft.
\ No newline at end of file
+#### GCP Private Service Connect
+
+[GCP Private Service Connect](https://cloud.google.com/vpc/docs/private-service-connect) allows you to open a path to Temporal without opening a public egress.
+It establishes a private connection between your Google Virtual Private Cloud (VPC) and Temporal Cloud.
+This one-way connection means Temporal cannot establish a connection back to your service.
+This is useful if normally you block traffic egress as part of your security protocols.
+If you use a private environment that does not allow external connectivity, you will remain isolated.
+
+Set up Private Service Connect with Temporal Cloud with these steps:
+
+1. Open the Google Cloud console
+2. Navigate to _Network Services_, then _Private Service Connect_. If you haven't used _Network Services_ recently, you might have to find it by clicking on _View All Products_ at the bottom of the left sidebar.
+
+   ![GCP console showing Network Services, and the View All Products button](/img/gcp-private-service-connect/gcp-console.png)
+
+3. Go to the _Endpoints_ section. Click on _Connect endpoint_.
+
+   ![GCP console showing the endpoints, and the Connect endpoint button](/img/gcp-private-service-connect/connect-endpoint-button.png)
+
+4. Under _Target_, select _Published service_, this will change the contents of the form to allow you to fill the rest as described below
+
+   ![GCP console showing the endpoints, and the Connect endpoint button](/img/gcp-private-service-connect/connect-endpoint.png)
+
+  - For _Target service_, fill in the _Service name_ with the Private Service Connect Service Name for the region you’re trying to connect to:
+
+      | Region                 | Private Service Connect Service Name                                            |
+      | ---------------------- | ------------------------------------------------------------------------------- |
+      | `us-central1`          | `projects/PROJECT/regions/us-central1/serviceAttachments/temporal-api`          |
+      | `australia-southeast1` | `projects/PROJECT/regions/australia-southeast1/serviceAttachments/temporal-api` |
+
+  - For _Endpoint name_, enter a unique identifier to use for this endpoint. It could be for instance `temporal-api` or `temporal-api-<namespace>` if you want a different endpoint per namespace.
```

Comment:
```suggestion
  - For **Endpoint name**, enter a unique identifier to use for this endpoint. It could be for instance `temporal-api` or `temporal-api-<namespace>` if you want a different endpoint per namespace.
```